Modules/booking/Model/BkNightWE.php

```diff
@@ -24,6 +24,21 @@ public function __construct() {
         $this->primaryKey = "id";
     }
 
+    public function getDefault() {
+        return array(
+            "id" => "",
+            "id_belonging" => 0,
+            "tarif_unique" => 1,
+            "tarif_night" => 0,
+            "night_start" => 19,
+            "night_end" => 8,
+            "id_category" => 0,
+            "tarif_we" => 0,
+            "choice_we" => "",
```

Comment:
not the same as invoice defaults with saturday and sunday

Reply:
? when not set, seems to be handled as saturday + sunday....

Reply:
in invoice, this is set

Reply:
maybe get the areascheduling